pkgs/development/python-modules/opnieuw/default.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  typing-extensions,
+}:
+
+buildPythonPackage rec {
+  pname = "opnieuw";
+  version = "3.0.0";
```

```suggestion
  version = "3.0.0";
  pyproject = true;
```
Even if the package doesn't have a pyproject.toml, it is recommended to use the pyproject builder.